### PR TITLE
Minor syndicate tweak

### PR DESCRIPTION
**Pull request by kilkun**
Adds dehydrated assistant kits to syndicate uplinks. A child of dehydrated carp, it spawns greytide SNPCs. You get 5 in a bundle that costs 15tc. Useful for a distraction, not much else.

The syndicate soft suits now have no slowdown, giving reason to actually use them instead of completely disregarding them.

:cl:
add: Added new item to syndicate uplinks.
tweak: small buff to syndicate soft suit
/:cl:

### CODE REVIEW

**Comment by anconfuzedrock** [comment]
you should probably remove the armor on those soft suits too if you do this, its as good as nuke op armor in most aspects

File: code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm [hunk 1]
```diff
@@ -133,6 +133,18 @@
 				SetScore(BOSS_SCORE,C,1)
 				SetScore(score_type,C,1)
 
+//NO MORE MEGAFAUNA ON THE STATION YOU SHITTERS
+/mob/living/simple_animal/hostile/megafauna/process()
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm]: this is why you don't work off master

File: code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm [hunk 2]
```diff
@@ -133,6 +133,18 @@
 				SetScore(BOSS_SCORE,C,1)
 				SetScore(score_type,C,1)
 
+//NO MORE MEGAFAUNA ON THE STATION YOU SHITTERS
```
Comment (lzimann) [inline code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm]: nice try to sneak it in

Comment (kilkun) [inline code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm]: oops

**Comment by kilkun** [comment]
you get nuke ops RIGs for free and one soft suit.

it has less armor, less heat protection and isn't insulated so even with your combat gloves it leaves you vulnerable to tesla arcs

File: code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm (hunk 1 above)
Comment (kilkun) [inline code/modules/mob/living/simple_animal/hostile/megafauna/megafauna.dm]: fixed

**Comment by PKPenguin321** [comment]
The snpc assistants, while cool, are really buggy and not ready to be added as anything more than an occasional admin gimmick

**Comment by kilkun** [comment]
ahem I addressed that:
_**Rejected** androids developed for a secret initiative_

**Comment by Haevacht** [comment]
That's not sufficient. And your PR title is wrong. And your megafauna shit is still there.

**Comment by kilkun** [comment]
no its not, this PR never got reopened thus it doesn't show the commit where i removed that

go comment on the new one

**Pull request closed without merging**